tests/data/nist/lj-1.toml

```diff
@@ -0,0 +1,6 @@
+[input]
+version = 1
+
+[[systems]]
+file = "configuration/lj-3.xyz"
```

Comment:
Thanks ! My bad I don't know why I thought this was the third dataset, it's clear in the errors that it's the first.